core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -160,7 +160,9 @@ abstract class RawModule extends BaseModule {
           // probes have their refs set eagerly
           case _ => // don't name literals
         }
-      } // else, don't name unbound types
+      }
+    case m: SramTarget =>
+      id.forceName(default = "_MEM", _namespace)
```

Comment:
```suggestion
      id.forceName(default = "MEM", _namespace)
```
This is not related to my comment below because if you do that, we should never actually hit this default, but we should just default to `MEM` (like `MemBase` above)